quickjs.h

```diff
@@ -845,8 +846,41 @@ typedef enum JSTypedArrayEnum {
     JS_TYPED_ARRAY_FLOAT64,
 } JSTypedArrayEnum;
 
+static inline int JS_BytesPerElement(JSTypedArrayEnum type)
+{
+    switch (type) {
+    case JS_TYPED_ARRAY_UINT8C:
+    case JS_TYPED_ARRAY_INT8:
+    case JS_TYPED_ARRAY_UINT8:
+        return 1;
+    case JS_TYPED_ARRAY_INT16:
+    case JS_TYPED_ARRAY_UINT16:
+        return 2;
+    case JS_TYPED_ARRAY_INT32:
+    case JS_TYPED_ARRAY_UINT32:
+    case JS_TYPED_ARRAY_FLOAT32:
+        return 4;
+    default:
+        break;
+    }
+    return 8;
+}
```

Comment:
Why not `default: return 8;` ?

Reply:
I encountered several times in the past, especially with old compiles, that they complain about that function has no return statement after the switch statement.

Of two choices I prefer

```
default:
        return 8;
```

because return 0 should never happen and we avoid extra comparison checks.